Makefile

```diff
@@ -374,7 +374,7 @@ MOCKOBJECTS = $(MOCK_LIB_SOURCES:.cc=.o)
 
 GTEST = $(GTEST_DIR)/gtest/gtest-all.o
 TESTUTIL = ./util/testutil.o
-TESTHARNESS = ./util/testharness.o $(TESTUTIL) $(MOCKOBJECTS) $(GTEST)
+TESTHARNESS = ./util/testharness.o $(TESTUTIL) $(MOCKOBJECTS) $(LIBOBJECTS) $(GTEST)
```

Comment:
I don't fully understand this change. 
Does this mean ldb's code is not checked as party of unity?
If its already checked, we don't have to do this again for trace_analyzer I think.

Reply:
Looks like it's not linked correctly right now. The Makefile change is to fix the following:
> tools/trace_analyzer_tool.cc:966: error: undefined reference to 'rocksdb::LDBCommand::HexToString(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
tools/trace_analyzer_tool.cc:995: error: undefined reference to 'rocksdb::LDBCommand::StringToHex(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'

Reply:
what's the make command that fails with that error?

Reply:
> /mnt/gvfs/third-party2/gcc/112ec378fec7002ad3e09afde022e656049f7191/5.x/centos7-native/c447969/bin/g++ db/db_test.o db/db_test_util.o util/testharness.o util/testutil.o table/mock_table.o util/fault_injection_test_env.o third-party/gtest-1.7.0/fused-src/gtest/gtest-all.o unity.a /mnt/gvfs/third-party2/jemalloc/b1a0e56c1e3e6929813a4331ade3a58ff083afbb/master/gcc-5-glibc-2.23/aa64d6b/lib/libjemalloc.a /mnt/gvfs/third-party2/snappy/7f9bdaada18f59bc27ec2b0871eb8a6144343aef/1.1.3/gcc-5-glibc-2.23/9bc6787/lib/libsnappy.a  /mnt/gvfs/third-party2/zlib/22c2d65676fb7c23cfa797c4f6937f38b026f3cf/1.2.8/gcc-5-glibc-2.23/9bc6787/lib/libz.a  /mnt/gvfs/third-party2/bzip2/dc49a21c5fceec6456a7a28a94dcd16690af1337/1.0.6/gcc-5-glibc-2.23/9bc6787/lib/libbz2.a  /mnt/gvfs/third-party2/lz4/907b498203d297947f3bb70b9466f47e100f1873/r131/gcc-5-glibc-2.23/9bc6787/lib/liblz4.a  /mnt/gvfs/third-party2/zstd/af6628a46758f1a15484a1760cd7294164bc5ba1/1.3.5/gcc-5-glibc-2.23/03859b5/lib/libzstd.a  /mnt/gvfs/third-party2/gflags/0b9929d2588991c65a57168bf88aff2db87c5d48/2.2.0/gcc-5-glibc-2.23/9bc6787/lib/libgflags.a  /mnt/gvfs/third-party2/numa/9cbf2460284c669ed19c3ccb200a71f7dd7e53c7/2.0.11/gcc-5-glibc-2.23/9bc6787/lib/libnuma.a /mnt/gvfs/third-party2/tbb/ff4e0b093534704d8abab678a4fd7f5ea7b094c7/2018_U5/gcc-5-glibc-2.23/9bc6787/lib/libtbb.a -B/mnt/gvfs/third-party2/binutils/55031de95a2b46c82948743419a603b3d6aefe28/2.29.1/centos7-native/da39a3e/bin/gold -Wl,--dynamic-linker,/usr/local/fbcode/gcc-5-glibc-2.23/lib/ld.so /mnt/gvfs/third-party2/libunwind/bf3d7497fe4e6d007354f0adffa16ce3003f8338/1.3/gcc-5-glibc-2.23/b443de1/lib/libunwind.a -Wl,-rpath=/usr/local/fbcode/gcc-5-glibc-2.23/lib -ldl -o unity_test  /mnt/gvfs/third-party2/lua/f0cd714433206d5139df61659eb7b28b1dea6683/5.2.3/gcc-5-glibc-2.23/65372bd/lib/liblua.a -L /mnt/gvfs/third-party2/libgcc/92b0c8e5c8eecc71eb042594ce1ab3413799b385/5.x/gcc-5-glibc-2.23/339d858/lib  -L /mnt/gvfs/third-party2/glibc/3d8698d5973ba94f41620a80a67e4457fdf01e90/2.23/gcc-5-glibc-2.23/ca1d1c0/lib -L /mnt/gvfs/third-party2/gcc/112ec378fec7002ad3e09afde022e656049f7191/5.x/centos7-native/c447969/lib64 -lgcc -lstdc++ -lpthread -lrt